packages/utilities/src/utilities/settings/settings.js

```diff
@@ -11,11 +11,11 @@
  * @exports ibmdotcom.settings
  * @type {object} Settings object
  * @property {string} [prefix=dds]
- * Carbon for IBM.com v1.41.1',
+ * Carbon for IBM.com v1.41.0-rc.0',
  *
  */
 const settings = {
-  version: 'Carbon for IBM.com v1.41.1',
+  version: 'Carbon for IBM.com v1.41.0-rc.0',
```

Comment:
it should be just `Carbon for IBM.com v1.41.0`, we don't update this settings file for every RC released

Reply:
currently taking bets on how many more commits it'll take me to get this right 😂